.github/dependabot.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+updates:
+- package-ecosystem: maven
+  directory: "/"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
+  reviewers:
+  - jglick
+  - oleg-nenashev
+  - batmat
+  ignore:
+  - dependency-name: jcl-over-slf4j
+    versions:
+    - ">= 0"
+  - dependency-name: log4j-over-slf4j
```

Comment:
We should specify the `groupId` too I guess?

Reply:
might be, though name conflict is unlikely